frontend/server/src/Controllers/QualityNomination.php

```diff
@@ -113,7 +137,8 @@ public static function apiCreate(\OmegaUp\Request $r) {
                 throw new \OmegaUp\Exceptions\InvalidParameterException('parameterInvalid', 'contents');
             }
             // Tags must be strings.
-            if (isset($contents['tags']) && is_array($contents['tags'])) {
+            if (isset($contents['tags'])) {
```

Comment:
```suggestion
            if (isset($contents['tags'])) {
                if (!is_array($contents['tags'])) {
                    throw new \OmegaUp\Exceptions\InvalidParameterException(
                        'parameterInvalid',
                        'contents'
                    );
                }
```

el test de `is_array()` sí era necesario. pero mejor hay que arrojar una excepción si no es un arreglo en vez de ignorarlo.

Reply:
Es que esa verificación ya se hace en `L123`(https://github.com/omegaup/omegaup/pull/2858/files/834e30727a0322a7997005048819b51ddbcc0c4b#diff-3416e72bd5e5e81d6c3d9169312e8804R123). Aquí correspondería solo verificar si `isset(...)`. A no ser que me esté equivocando o.O

Reply:
no, tienes razón.